packages/host/app/components/ai-assistant/message/index.gts

```diff
@@ -240,3 +244,55 @@ export class AiAssistantConversation extends Component<AiAssistantConversationSi
     </style>
   </template>
 }
+
+interface CardPillSignature {
+  Element: HTMLDivElement;
+  Args: {
+    card: CardDef;
+  };
+  Blocks: {
+    default: [];
+  };
+}
+
+export class CardPill extends Component<CardPillSignature> {
```

Comment:
There's actually another `CardPill` component in `components/ai-assistant/card-picker/index`. That is the one we're using when cards are attached in the chat input area of the ai-assistant panel. It would be good if we could consolidate them. The difference would be to not to display the remove button. The styles should be the same otherwise.

Reply:
👍🏻 thanks, I’ve merged them now successfully 🎉

<img width="387" alt="Boxel 2024-03-14 11-06-46" src="https://github.com/cardstack/boxel/assets/43280/cc33f7b2-3421-4213-9ddc-fd0c3e358680">